test.sh

```diff
@@ -121,6 +121,7 @@ test_unit()
 {
     echo "Testing Numba JIT Compiled Functions"
     pytest -rsx -W ignore::RuntimeWarning -W ignore::DeprecationWarning tests/test_gpu_stump.py
+    check_errs $?
```

Comment:
I purposely do NOT check errors for GPU. I don't think we should add this

Reply:
I will remove this one and the similar ones you commented below.